### DISCUSSION TITLE
cSpell.enableFiletypes for semicolon CSV?

### DISCUSSION THREAD

**Discussion by metablaster** [streetsidesoftware/vscode-spell-checker#2346] [Q&A] [answered]
My settings for this were:

```
	"cSpell.enableFiletypes": [
		"ignore",
		"jsonc",
		"json",
		"properties",
		"log",
		"csv (semicolon)",
		"ini"
	],
```

Problem is with `"csv (semicolon)"` above because it's flagged as invalid, I assume there is another keyword for CSV files?
How do I specify csv for semicolon and also how to for coma separated CSV?

**Comment by Jason3S** [comment, marked as answer]
@metablaster,

Even though the schema complains, it should still work. You might need to restart VS Code to get it to understand a new file type.

<img width="900" alt="image" src="https://user-images.githubusercontent.com/3740137/201963410-c011b5e2-974c-428c-aae3-ac05646bb7eb.png">

**Reply by metablaster** [reply]
Thanks Jason!
I can now put a comment into my setting s file about false flag and rest in peace :)

**Comment by metablaster** [comment]
Btw. I figured out something with little testing...

If you set `cSpell.enableFiletypes` in `settings.json` it's flagged, but if you put it into `cspell.json` it's just fine.

Also some keywords are not recognized in `cspell.json` and must be put into `settings.json` this forces you to split your settings accros files, which with a bit of trial and error all works out fine.
